frontend/src/app/organizations/organizations-list/organizations-list.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { Location } from '@angular/common';
+import { DebugElement } from '@angular/core';
+import { async, ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { OrganizationsListComponent } from './organizations-list.component';
+
+describe('OrganizationsComponent', () => {
+  let component: OrganizationsListComponent;
+  let fixture: ComponentFixture<OrganizationsListComponent>;
+  let debugElement: DebugElement;
+  let location: Location;
+  const organizations = [{
+      address: 'Winterfell, ul. Przed Murem 1',
+      description: 'Winter is comming',
+      id: 93,
+      name: 'Starks Family',
+      slug: 'stark-family',
+      url: 'http://localhost:4200/api/organizations/93/'
+    }, {
+      address: 'Casterly Rock',
+      description: 'A Lannister always pays his debts.',
+      id: 94,
+      name: 'Lannister Family',
+      slug: 'lannister-family',
+      url: 'http://localhost:4200/api/organizations/94/',
+    }, ];
+  beforeEach(async(() => {
```

Comment:
that's not a crime to use empty lines between tests definitions - it will became a lot easier to be read.